releasenotes/notes/release-train-02-2023-package-refresh-9de8049f8cb08a5e.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Package repositories and container images for CentOS Stream based deployments have been updated.
+    Key packages to note are:
+
+    * Kernel
+      * version: 4.18.0
+      * release: 408.el8
+    * Libvirt
+      * version: 7.6.0
```

Comment:
The nova-libvirt container images we built recently were all using Libvirt 8.0.0. Did you change the configuration to get Libvirt from somewhere else?